src/main/java/com/github/platymemo/alaskanativecraft/client/renderer/item/HarpoonItemRenderer.java

```diff
@@ -13,10 +13,13 @@
 import net.minecraft.item.ItemStack;
 import net.minecraft.util.math.Vec3f;
 
-public enum HarpoonItemRenderer {
-    INSTANCE;
+public class HarpoonItemRenderer {
```

Comment:
Why was this switched to a class?

Reply:
I am not sure anymore, probably was necessary at some point and never got changed back. Reverted in 6b269557a7c3b92c487bc0445a85b4bf9bfa86a0